stdlib/config-api/src/main/java/org/ballerinalang/stdlib/config/GetConfig.java

```diff
@@ -65,16 +65,27 @@ public static Object get(String configKey, String type) {
     @SuppressWarnings("unchecked")
     private static MapValue buildMapValue(Map<String, Object> section) {
         MapValue map = new MapValueImpl<String, Object>();
-
-        section.forEach((key, val) -> {
-            if (val instanceof String || val instanceof Long || val instanceof Double || val instanceof Boolean) {
-                map.put(key, val);
-            }
-        });
+        section.forEach((key, val) -> map.put(key, getConvertedValue(val)));
```

Comment:
Shall we replace this with a normal for loop? We now avoid the use of these as much as possible and reverting the current usages back to normal loops

Reply:
+1.

Reply:
Corrected.